reactor-core/src/main/java/reactor/core/publisher/Operators.java

```diff
@@ -2310,11 +2325,19 @@ final void drainLoop() {
 		final T value;
 
 		volatile int once;
+		
+		String stepName;
```

Comment:
this field can be final (and as such should be moved above the `volatile` one)

Reply:
done